<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN487105" failures="1" tests="4" name="edge-Test" time="4083.305" errors="0" timestamp="2024-04-02T13:23:39 IST">
  <testcase classname="testCases.TC_001_AroundCognizant" name="testUser" time="24.375"/>
  <testcase classname="testCases.TC_001_AroundCognizant" name="checkNewsToolTip" time="15.907"/>
  <testcase classname="testCases.TC_001_AroundCognizant" name="checkAllNews" time="12.498"/>
  <testcase classname="testCases.TC_001_AroundCognizant" name="lastTest" time="112.032">
    <failure type="java.io.FileNotFoundException" message="C:\Users\2318412\eclipse-workspace\CAS\TestData\NewsAround_Cognizant.xlsx (The process cannot access the file because it is being used by another process)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\2318412\eclipse-workspace\CAS\TestData\NewsAround_Cognizant.xlsx (The process cannot access the file because it is being used by another process)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
at utilities.ExcelUtility.excelUtility(ExcelUtility.java:19)
at utilities.ExcelUtility.output(ExcelUtility.java:25)
at testCases.TC_001_AroundCognizant.lastTest(TC_001_AroundCognizant.java:236)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- lastTest -->
</testsuite> <!-- edge-Test -->
